.github/workflows/docker-image.yml

```
@@ -0,0 +1,18 @@
+name: Docker Image CI
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
```

```suggestion
workflow_dispatch:
```

Der wäre noch sehr cool, um für den Fall die Action per Hand zu triggern ^^